jwst/assign_wcs/util.py

```diff
@@ -883,10 +883,22 @@ def compute_footprint_spectral(model):
 
     x, y = grid_from_bounding_box(bbox)
     ra, dec, lam = swcs(x, y)
-    footprint = np.array([[np.nanmin(ra), np.nanmin(dec)],
-                          [np.nanmax(ra), np.nanmin(dec)],
-                          [np.nanmax(ra), np.nanmax(dec)],
-                          [np.nanmin(ra), np.nanmax(dec)]])
+
+    # the wrapped ra values are forced to be on one side of ra-boarder
+    # the wrapped ra are used to determine the correct  min and max ra
+    ra = wrap_ra(ra)
+    min_ra = np.nanmin(ra)
+    max_ra = np.nanmax(ra)
+
+    # for the footprint we want the ra values to fall between 0 to 360
+    if(min_ra < 0):
+        min_ra = min_ra + 360.0
+    if(max_ra > 360.0):
```

Comment:
line 894 allows for RA=0.0 (it only disallows RA < 0), so should RA=360.0 be disallowed, since it's redundant with RA=0?  In other words, should this perhaps be `if (max_ra >= 360.0)"